utils/bin/lvim

```diff
@@ -3,4 +3,36 @@
 export LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-$HOME/.local/share/lunarvim}"
 export LUNARVIM_CONFIG_DIR="${LUNARVIM_CONFIG_DIR:-$HOME/.config/lvim}"
 
+headless_update() {
+  nvim --cmd 'lua _G.__lvim_updating=true' -u "$LUNARVIM_RUNTIME_DIR/lvim/init.lua" --noplugin -n -i NONE --headless
+}
+
+for arg; do
+  shift
+  case $arg in
+    --update)
+      :
+      echo 'Updating LunarVim...'
+      old="$(pwd)"
+      cd "$LUNARVIM_RUNTIME_DIR/lvim"
```

Comment:
You don't need to change the directory if you only need it for `git`, instead you can use
```console
git -C $LVIM_BASE_DIR status
```

Reply:
Doing that for every command is tedious, but I guess we could? I'm not sure why cd is bad here.